src/electos/ballotmaker/ballots/ballot_layout.py

```diff
@@ -27,6 +26,10 @@
 )
 from reportlab.platypus.flowables import CondPageBreak
 
+logging.getLogger(__name__)
+
+CANDIDATE = "candidate"
+BALLOT_MEASURE = "ballot measure"
```

Comment:
FYI: [electos.ballotmaker.data.models.ContestType enum](https://github.com/TrustTheVote-Project/BallotLab/blob/1e5b7ffe/src/electos/ballotmaker/data/models.py#L61) has the canonical value of both these strings.

```python
class ContestType(Enum):

    BALLOT_MEASURE = "ballot measure"
    CANDIDATE = "candidate"
```

Reply:
I saw your Enum code but was too pressed for time to incorporate it in my code. Let me know when the Enum class ands in the module or class where you think it belongs and I'll use it moving forward.

I'll create an issue to track this, too

Reply:
`ContestType` is already in `development`.

See: [electos.ballotmaker.data.models.ContestType enum](https://github.com/TrustTheVote-Project/BallotLab/blob/1e5b7ffe/src/electos/ballotmaker/data/models.py#L61)